### DISCUSSION TITLE
SAE Port 8080 wechseln

### DISCUSSION THREAD

**Discussion by RobToas** [camueller/SmartApplianceEnabler#537] [Installation und Betrieb]
Hallo,
Gibt es eine einfache Möglichkeit, das Port 8080 von SAE zu ändern?
(ie http://192.168.xxx.xxx:8080/)
Danke für die Rückmeldung!
Rob

**Comment by camueller** [comment]
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#sae5
